ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ImmutableDB/TestBlock.hs

```diff
@@ -274,14 +276,22 @@ shrinkCorruptions :: Corruptions -> [Corruptions]
 shrinkCorruptions =
   fmap (NE.fromList . getNonEmpty) . shrink . NonEmpty . NE.toList
 
-
 {-------------------------------------------------------------------------------
   Top-level tests
 -------------------------------------------------------------------------------}
 
-
 tests :: TestTree
 tests = testGroup "TestBlock"
     [ testProperty "TestBlock Binary roundtrip"   prop_TestBlock_Binary
     , testProperty "testBlockEpochFileParser"     prop_testBlockEpochFileParser
     ]
+
+{-------------------------------------------------------------------------------
+  Orphans
+-------------------------------------------------------------------------------}
+
+instance Arbitrary FsPath where
+  arbitrary = undefined
```

Comment:
?

Reply:
We only use the `shrink` here. It's just for this particular test.

Reply:
Use `error` with a comment/message stating that, because `undefined` looks like something temporary that was left in by accident.

Reply:
Fair enough :)